scripts/python-skywater-pdk/skywater_pdk/generate-resources-rst.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 SkyWater PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import argparse
+from pathlib import Path
+import pandas as pd
+
+
+def parse_entries(filepath, sheet_name, cols):
+    values = pd.read_excel(
```

Comment:
Is there an OpenDocumentFormat for spreadsheets we can use instead?

Reply:
`read_excel` method:
```
Supports xls, xlsx, xlsm, xlsb, odf, ods and odt file extensions read from a local filesystem or URL. Supports an option to read a single sheet or a list of sheets.
```
we can use ODS here.

Reply:
I switched to using ODS files